.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,9 +6,14 @@ resolves #`issueNumber`
 
 - [ ] You've followed the code structure 
 - [ ] You've linked the PR to correct Issue
+- [ ] Passes the CI pipeline tests (lint, formatting, other ones)
 
 ## Proposed Changes
 
   -
   -
-  -
\ No newline at end of file
+  -
+
+## Additonal Info
+
+### You can attach screenshots here, updated dependencies (if any), any breaking changes, etc.
```

Comment:
The description for additional info will be commented out, not a third level heading